source/app.d

```diff
@@ -55,6 +55,21 @@ shared static this()
 	immutable certPath = redhatCA.exists ? redhatCA : debianCA;
 }
 
+// generate dummy data for e.g. Heroku's preview apps
+void defaultInit(UserManController userMan, DubRegistry registry)
+{
+	if (environment.get("GENERATE_DEFAULT_DATA", "0") == "1" &&
+		registry.getPackageDump().empty)
+	{
+		logInfo("'GENERATE_DEFAULT_DATA' is set and an empty database has been detected. Inserting dummy data.");
+		auto userId = userMan.registerUser("dummy@dummy.org", "dummyUser",
+			"Dummy User", "test1234");
+		DbRepository repo;
+		repo.parseURL(URL("https://github.com/libmir/mir-optim"));
+		registry.addPackage(repo, userId);
```

Comment:
We can add a few more once this is working (though we probably can't add too much due to the restriction of the unauthorized GitHub API).